framework/auth/oauth_scopes.py

```diff
@@ -62,9 +72,9 @@ class ComposedScopes(object):
                     (CoreScopes.NODE_BASE_WRITE, CoreScopes.NODE_CHILDREN_WRITE, CoreScopes.NODE_LINKS_WRITE)
 
     # Privileges relating to editing content uploaded under that node # TODO: Add wiki etc when implemented
-    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ,)
+    NODE_DATA_READ = (CoreScopes.NODE_FILE_READ, CoreScopes.NODE_COMMENTS_READ)
```

Comment:
I'm on the fence about whether to include this in:

a) Node metadata (the ability to see a project is the ability to comment on it),
b) Node data (you must have special permission to affect the content of a project- eg files access, before being able to participate in a discussion)
c) A third (possibly new) scope

Any opinions?